src/content-handlers/iiif/modules/uv-contentleftpanel-module/css/styles.less

```diff
@@ -156,3 +156,9 @@
     }
   }
 }
+@-moz-document url-prefix() {
+  .iiif-gallery-component .header input[type="range"] {
+    margin-top: 7px;
+    width: 300px;
+  }
+}
```

Comment:
Does this make any difference?

```
.iiif-gallery-component .header input[type=range]::-moz-range-track {
  width: 100px;
  z-index: -1;
}
```

(I haven't tested, just wondering if it might help; I took inspiration from [this StackOverflow](https://stackoverflow.com/questions/30723341/different-z-index-for-thumb-and-track-in-html5-range-slider-in-firefox)...)

Reply:
It does! Thanks @demiankatz. The margin-top property now doesn't work though so I'll try and fix that to move the bar up a bit more 
<img width="192" alt="Screenshot 2024-10-29 at 11 27 54" src="https://github.com/user-attachments/assets/4f084259-15fd-4a6b-835b-b7c174dd9ebf">

Reply:
Excellent, thanks! Since the width part works, it looks like you might be able to drop the z-index line to simplify further.

Reply:
Done that, and since margin-top wasn't working with the bar I added it to the buttons instead to move them down so it looks better

Reply:
Looks reasonable to me, but I'm not seeing the focus highlighting here; I suspect this branch may have been created before the other PR was merged. Do you mind merging the upstream dev branch into this so we can look at all the fixes together in one place?